docs/code_overview/other.rst

```diff
@@ -15,6 +15,13 @@ instances of them bound to an attribute of one of the PRAW models.
 
 .. toctree::
    :maxdepth: 2
+   :caption: Reddit Live Helpers
+
+   other/livecontributorrelationship
+   other/livethreadcontribution
+
```

Comment:
I added this new section because I thought it's not appropriate to put LiveThreadContribution into 'Moderation Helpers' section.